Commands/ImportGA4Reports.php

```diff
@@ -38,6 +38,7 @@ protected function configure()
         $this->setName('googleanalyticsimporter:import-ga4-reports');
         $this->setDescription('Import GA4 reports from one or more google analytics properties into Matomo sites.');
         $this->addRequiredValueOption('property', null, 'The GA properties to import.');
+        $this->addRequiredValueOption('streamId', null, 'The streamID for which you want to pull data, for multiple value send comma separated streamIds, by default it pulls data for all the streamIds');
```

Comment:
This would probably work better as multiple sentences. What about?
`The streamId for which you want to pull data. For multiple streams, use comma separated streamIds. By default it pulls data for all streams.`
What do you think @Stan-vw ?

Reply:
@snake14 Updated as suggested